xcode/Subconscious/Shared/Components/AddressBook/AddressBookModel.swift

```diff
@@ -36,40 +41,50 @@ struct FollowUserFormModel: Equatable {
 }
 
 struct PetnameFieldCursor: CursorProtocol {
-    static func get(state: AddressBookModel) -> FormField<String, Petname> {
+    typealias Model = AddressBookModel
```

Comment:
These changes were byproducts of experimenting with type-erased generics. I ended up going back to concrete types for services, but the typealiases may as well stay. It's cleaner.